.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,44 @@
+name: 🐞 Bug Report
+description: Report an issue
+labels: ["awaiting-triage", "bug 🐞"]
```

Comment:
Is this referring to GitHub labels? If yes, we currently don't have these labels, so let's drop this line?

Reply:
removed.